src/types/onyx/ReportAction.ts

```diff
@@ -32,6 +32,7 @@ type Message = {
     isEdited: boolean;
 
     isDeletedParentAction: boolean;
+    isReversedTransaction?: boolean;
```

Comment:
We were discussing [here](https://expensify.slack.com/archives/C03TQ48KC/p1696824818157629) to add description for all of these, do you mind adding one as well? Thanks!